classes/item.rb

```diff
@@ -0,0 +1,48 @@
+class Item
+  attr_accessor :publish_date, :archived, :label, :author, :genre, :source
+  attr_reader :id
+
+  def initialize(publish_date, archived: false)
+    @id = rand(1..1_000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  # def add_genre(genre)
+  #   @genre = genre
+  #   # genre.add_item(self)
+  # end
+
+  # def add_author(author)
+  #   @author = author
+  #   # author.add_item(self)
+  # end
+
+  # def add_label(label)
+  #   @label = label
+  #   # label.add_item(self)
+  # end
```

Comment:
Kindly remove the commented-out code to keep the code base clean. Remember, it is super important not to leave console statements or commented code in our code base.